client/python/apache_polaris/cli/command/catalogs.py

```diff
@@ -76,6 +76,7 @@ class CatalogsCommand(Command):
     sts_endpoint: str
     sts_unavailable: bool
     path_style_access: bool
+    user_token_sts: bool
```

Comment:
nit: Combining CLI changes and backend changes in the same PR is ok, but these areas usually attract different sets of reviewers. As for me, I usually do CLI changes in a follow-up PR. I also usually do not usually have material opinions about CLI code changes :) all in all, it might be easier for reviewers to have CLI and java changes in different PRs... I guess.